community/kernel/src/main/java/org/neo4j/kernel/impl/index/labelscan/NativeLabelScanStore.java

```diff
@@ -180,7 +180,7 @@ public NativeLabelScanStore( PageCache pageCache, File storeDir, FullStoreChange
         this.pageCache = pageCache;
         this.pageSize = pageSize;
         this.fullStoreChangeStream = fullStoreChangeStream;
-        this.storeFile = new File( storeDir, FILE_NAME );
+        this.storeFile = getLabelScanStoreFile( storeDir );
```

Comment:
It's not actually considered a store. Not found in `StoreType` or in `StoreFile`.
Maybe it _does_ make sense to _make_ it a store, but I would say that is outside the scope of this PR.